gwlfe/StreamBank.py

```diff
@@ -113,25 +109,31 @@ def CalculateStreamBankEros(z, Y):
 
         z.GroundPhos[Y][i] -= (((z.RCNMAC + z.HPNMAC) / z.AreaTotal) * z.GroundPhos[Y][i] * z.n78)
 
-        z.GroundNitr[Y][0] += z.GroundNitr[Y][i]
-        z.GroundPhos[Y][0] += z.GroundPhos[Y][i]
-
-        z.TileDrain[Y][0] += z.TileDrain[Y][i]
-        z.TileDrainN[Y][0] += z.TileDrainN[Y][i]
-        z.TileDrainP[Y][0] += z.TileDrainP[Y][i]
-        z.TileDrainSed[Y][0] += z.TileDrainSed[Y][i]
-        z.AnimalN[Y][0] += z.AnimalN[Y][i]
-        z.AnimalP[Y][0] += z.AnimalP[Y][i]
-
-        z.GRLostBarnN[Y][0] += z.GRLostBarnN[Y][i]
-        z.GRLostBarnP[Y][0] += z.GRLostBarnP[Y][i]
-        z.GRLostBarnFC[Y][0] += z.GRLostBarnFC[Y][i]
-        z.NGLostBarnN[Y][0] += z.NGLostBarnN[Y][i]
-        z.NGLostBarnP[Y][0] += z.NGLostBarnP[Y][i]
-        z.NGLostBarnFC[Y][0] += z.NGLostBarnFC[Y][i]
-        z.NGLostManP[Y][0] += z.NGLostManP[Y][i]
+        z.GroundNitrSum[Y] += z.GroundNitr[Y][i]
```

Comment:
Although it's a bit tricker to compare to the original code, I think making dedicated `*Sum` variables are a big improvement for reasoning about these than using the `0` index on the array, good call for the switch.